src/ui-config/marketsConfig.tsx

```diff
@@ -336,6 +337,27 @@ export const marketsData: {
       DEBT_SWITCH_ADAPTER: AaveV3Base.DEBT_SWAP_ADAPTER,
     },
   },
+  [CustomMarket.proto_arbitrum_sepolia_v3]: {
+    marketTitle: 'Arbitrum Sepolia',
+    market: CustomMarket.proto_arbitrum_sepolia_v3,
+    v3: true,
+    chainId: ChainId.arbitrum_sepolia,
+    enabledFeatures: {
+      faucet: false,
+      incentives: true,
```

Comment:
not sure we have incentives here for testnet

Reply:
I think it might make sense to just remove all of the enabledFeatures config for the new testnets, and then since the new ones don't have a faucet address, you could just link the external faucet if there isn't a faucet address. We probably need to go through and clean up the config a little bit because there's some things that aren't even used anymore, but we can tackle that later.